### PR TITLE
Add feature counts to the right branch

### PR DESCRIPTION
**Pull request by paylakatel**
This PR:
- adds feature counts back to the map
     - They are implemented a little bit differently than they previously were due to the old counts not displaying correctly when >4000. Now we query the layer after updating its display and return a list of feature ids. The length of that list populates the feature count (this.state.pointCount)
- adds a few functions that format the modeSelection, dates, and dataset selection so they are nicer to display in the feature counts
- makes the icons a little smaller as requested by the stakeholdersadds a readme

### CODE REVIEW

File: components/Map.js [hunk 1]
```diff
@@ -195,15 +209,50 @@ class Map extends React.Component {
...
+  };
+
+  // Make dataset selection nicer for displaying in FeatureCounts
+  formatDataSelection = dataSelection => {
+    return dataSelection == 'crash' ? 'crashes' : 'fatalities';
+  };
+
+  // Make selected dates nicer for displaying in FeatureCounts
+  formatDate = date => {
+    return format(date, 'MM/D/YY');
```
Comment (fionawhim) [inline components/Map.js:232]: Does `MM` give you "01" for January?

File: components/Map.js [hunk 2]
```diff
@@ -195,15 +209,50 @@ class Map extends React.Component {
...
   render() {
     return (
       <div>
         {/* make map take up entire viewport with room for the navbars */}
         <div
           style={{ height: 'calc(100vh - 125px)' }}
           ref={this.setMapEl}
-          lastUpdated={this.state.lastUpdatedDate}
-        />
+          lastupdated={this.state.lastUpdatedDate}
```
Comment (fionawhim) [inline components/Map.js]: What does the `lastupdated` attribute on a `<div>` do?

File: components/Map.js [hunk 3]
```diff
@@ -195,15 +209,50 @@ class Map extends React.Component {
...
+        >
+          <div
+            style={{ zIndex: 1000, position: 'relative', fontFamily: 'Lora' }}
+          >
+            <FeatureCounts
+              pointCount={this.state.pointCount}
```
Comment (fionawhim) [inline components/Map.js:244]: 👍

File: components/Map.js (hunk 1 above)
Comment (paylakatel) [reply components/Map.js:232]: yes

File: components/Map.js (hunk 2 above)
Comment (paylakatel) [reply components/Map.js]: nothing, that's a typo - thanks for finding

**Comment by paylakatel** [comment]
The items in this PR were addressed [here](https://github.com/CityOfBoston/vision-zero/pull/24)

**Pull request closed without merging**